location-service/src/main/java/uk/gov/cshr/locationservice/UK_NUTS.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.cshr.locationservice;
```

Comment:
This class should be in a util package or something similar not a top level class at the same level as the entry point.